functions-and-scope.js

```diff
@@ -27,7 +35,22 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // cumLaude([6, 4, 5]) geeft 0
 // cumLaude([8, 9, 4, 6, 10]) geeft 3
 
+function cumLaude(list) {
+    let counter = 0;
+    for (let i = 0; i < list.length; i++) {
+        if (list[i] >= 8) {
+            counter = counter + 1;
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
            counter++;
```